src/airport.js

```diff
@@ -18,26 +17,29 @@ class Airport {
     if (this.flightNumber === this.planeToLand) {
       this.landingPlane = true;
     }
-  
   }
 
-  getCapacity(){
+  getCapacity() {
+    this.defaultCapacity = 0;
     return this.defaultCapacity;
   }
 
   changeCapacity() {
+    this.newCapacity = 10;
     if (this.newCapacity) {
       this.defaultCapacity = this.newCapacity
     }
   }
   
-  atDefaultCapacity () {
-    if (this.airportPlanes.length == this.defaultCapacity || this.airportPlanes.length == this.newDefaultCapacity ) {
+  atDefaultCapacity() {
+    this.defaultCapacity = 0;
+    if (this.airportPlanes.length == this.defaultCapacity || this.airportPlanes.length == this.newCapacity ) {
       this.airportFull = true;
        } 
   }
   
   preventAirportPush() {
+    this.airportFull = true
```

Comment:
probaly should have written this in a different way so that I didn't have to define the airportFull within the function